### DISCUSSION TITLE
Disable Whatsapp Log / Debug

### DISCUSSION THREAD

**Discussion by ridhodo** [tulir/whatsmeow#532] [General]
Is there a way to disable debug/log from WhatsApp's WebSocket? 
I have attempted to disable it by filling `container := sqlstore.NewWithDB(dbConf, "pgx", nil)` during initialization
Please inform me if there is a possible solution. Thankyou

**Comment by arugaz** [comment]
waLog [Noop](https://github.com/tulir/whatsmeow/blob/main/util/log/log.go#L34)
